src/components/Lightbox.tsx

```diff
@@ -123,4 +147,13 @@ const styles = StyleSheet.create({
     ...textStyle.body,
     color: theme.color.white,
   },
+  buttonsContainer: {
+    flexDirection: 'row',
+    marginVertical: theme.margin.large,
+    marginHorizontal: theme.margin.common,
+  },
+  buttonContainer: {
+    width: '50%',
```

Comment:
Dit gaat mis als we drie buttons hebben. Ik denk dat het beter is om dit te vervangen door `flex: 0.5`, dat gaat wel goed bij >2 buttons.